packages/web-react/docs/stories/examples/FileUploaderWithImageCrop.stories.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+// import { ComponentStory } from '@storybook/react';
+import { SpiritFileUploaderAttachmentProps } from '../../../src/types';
+import { useFileQueue } from '../../../src/components/FileUploader/useFileQueue';
+import FileUploader from '../../../src/components/FileUploader/FileUploader';
+import FileUploaderInput from '../../../src/components/FileUploader/FileUploaderInput';
+import FileUploaderList from '../../../src/components/FileUploader/FileUploaderList';
+import FileUploaderAttachment from '../../../src/components/FileUploader/FileUploaderAttachment';
+import { Button, Modal, ModalBody, ModalDialog, ModalFooter } from '../../../src/components';
+
+export default {
+  title: 'Examples/Compositions',
+};
+
+export const FileUploaderWithImageCrop = (args) => {
```

Comment:
I suggest choosing a better name for describing this example. I am aware that we were talking about image crop usage on Jobs.cz, but this example mainly demonstrates a Modal with Image. So `FileUploaderWithModalImagePreview`, for example, will be a better name. :-)

Reply:
The same goes for the functions/methods.

Reply:
function and methods were renamed as you suggested